docs/3.tutorials/fts/0-intro.md

```diff
@@ -0,0 +1,59 @@
+---
+id: introduction
+title: Fungible Tokens Zero to Hero
+sidebar_label: Introduction
+---
+
+> In this _Zero to Hero_ series, you'll find a set of tutorials covering every aspect of a fungible token (FT) smart contract.
+> You'll start by interacting with a pre-deployed contract and by the end you'll have built a fully-fledged FT smart contract that supports every extension of the standards.
+
+---
+
+## Prerequisites
+
+To complete these tutorials successfully, you'll need:
+
+- [Rust](/develop/prerequisites)
+- [A NEAR account](/concepts/basics/accounts/creating-accounts)
+- [NEAR-CLI](/tools/near-cli#setup)
```

Comment:
"A NEAR account" shall probably change to "NEAR Testnet Account" leading to wallet.testnet.near.org  at the moment creating-accounts page is a bit hard to understand in the concept of "i just need an testnet account to get going"